docs/user-guide/a3-02-reference-capability-atoms.md

```diff
@@ -0,0 +1,2329 @@
+---
+layout: user-guide
+---
+
+Capability Atoms
+============================
+
+### Sections:
+
+1. [Targets](#Targets)
+2. [Stages](#Stages)
+3. [Versions](#Versions)
+4. [Extensions](#Extensions)
+5. [Compound Capabilities](#Compound-Capabilities)
+6. [Other](#Other)
+
+<br>
+
+Targets
+----------------------
+*Capabilities to specify code generation targets (`glsl`, `spirv`...)*<br><br>
```

Comment:
don't use `<br>`, just two empty lines.

And it seems that there are too many extraneous empty lines here.